torch/distributed/rendezvous.py

```diff
@@ -3,6 +3,8 @@
 except ImportError:
     from urlparse import urlparse
 
+import torch
```

Comment:
if we just need `_six` here, shall we explicitly import _six?

Reply:
I think so.